tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
-envlist = py35, py38, flake8
```

Comment:
py3.5 added here intentionally after [this comment](https://github.com/edx/cc2olx/pull/1#discussion_r455983172) in https://github.com/edx/cc2olx/pull/2/commits/5ed1eaa00ebdb0bf5369ac9db88b036f377abbd7 as well as some type casts of path for functions that lack support of `pathlib`.

And indeed, `lms` container of the devstack has python with version `3.5.2`.